addons/html_builder/static/tests/drag_and_drop.test.js

```diff
@@ -0,0 +1,121 @@
+import { expect, test } from "@odoo/hoot";
```

Comment:
maybe move it tests/website_builder ?

Reply:
It depends, the normal drag and drop is supposed to be the universal one, only the grid mode is specific to website so maybe I could split the tests in two ?
But for now, nothing blocks the grid mode so :thinking:

Reply:
But you are using s_three_columns ? It s an website snippet no ?